### PR TITLE
Add HorizontalLine and VerticalLine to diagnostics

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
## Description

Added HorizontalLine and VerticalLine classes to src/diagnostics.  The second plot has 2 horizontal and 2 vertical lines plotted using these new classes.

image

## Dependencies

None

## Impact

None

Completes #117

### CODE REVIEW

File: src/eva/diagnostics/horizontal_line.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        # Create declarative plotting HorizontalLine object
+        # -------------------------------------------
+        self.plotobj = emcpy.plots.plots.HorizontalLine(yval)
+
+        # Get defaults from schema
+        # ------------------------
+        layer_schema = config.get('schema', os.path.join(return_eva_path(), 'defaults',
+                                  'horizontal_line.yaml'))
+        config = get_schema(layer_schema, config, logger)
+#        delvars = ['y', 'type', 'schema']
```
Comment (CoryMartin-NOAA) [inline src/eva/diagnostics/horizontal_line.py]: remove comment

**Review by CoryMartin-NOAA — COMMENTED** [review]
One very minor thing. Also can we add one or both of these to one of the test YAMLs so that it gets exercised?

File: src/eva/diagnostics/horizontal_line.py (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply src/eva/diagnostics/horizontal_line.py]: Comment removed.   Is adding lines to an existing plot sufficient or would you prefer a stand-alone example in the test yamls (yamlae? yamlii? what is the correct plural for yaml?)?

Comment (CoryMartin-NOAA) [reply src/eva/diagnostics/horizontal_line.py]: no need for a new test, just throw something in one of the YAMLs. Like add a line at 0 on a histogram or something.

Comment (EdwardSafford-NOAA) [reply src/eva/diagnostics/horizontal_line.py]: Done.  The result of `testIodaObsSpaceAmsuaN19.yaml`'s density plots is so:

image

**Review by CoryMartin-NOAA — APPROVED** [review]

File: src/eva/diagnostics/horizontal_line.py (hunk 1 above)
Comment (CoryMartin-NOAA) [reply src/eva/diagnostics/horizontal_line.py]: Looks good, thanks!!

**Comment by danholdaway** [comment]
Thanks for adding this

**Pull request merged**